actionview/lib/action_view/helpers/tags/file_field.rb

```diff
@@ -2,6 +2,17 @@ module ActionView
   module Helpers
     module Tags # :nodoc:
       class FileField < TextField # :nodoc:
+
+        def render
+          options = @options.stringify_keys
+          add_default_name_and_id(options)
+          if options.fetch("include_hidden", true)
+            tag("input", :name => options["name"], :type => "hidden", :value => "") + super
```

Comment:
Not sure if we should call `super`. We already know how to build the input at this point. We would already have called `add_default_name_and_id` and we don't need to handle `size` or `value` type. So I'd just call `tag('input', options)` having set the current type.